plugins/vli/fu-vli-pd-firmware.c

```diff
@@ -42,6 +42,8 @@ fu_vli_pd_firmware_validate_header(FuVliPdFirmware *self)
 		return TRUE;
 	if (self->vid == 0x208E)
 		return TRUE;
+	if (self->vid == 0x0B05)
+		return TRUE;
```

Comment:
I am wondering if this check on the VID is useful, maybe we remove all the VID checks?

Reply:
Agree. We use VID to locate the header in the bin file because some of VL101 and VL102 have different addresses. I checked fwupd's support list, VL101 and VL102 are not there, and we won't have new products for them. I will remove the check code and test again.